packages/utilities/psammead-storybook-helpers/package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "@bbc/psammead-storybook-helpers",
+  "version": "0.1.0",
```

```suggestion
  "version": "1.0.0",
```

Probably best practice to make this v1 unless we know there are immediate changes coming?

---

I'm not aware of any immediate modifications for this. I'm okay with publishing it as version 1. I'm going to proceed to commit the suggestion.